src/fa.c

```diff
@@ -32,6 +32,7 @@
 #include <config.h>
 #include <limits.h>
 #include <ctype.h>
+#include <regex.h>
```

Comment:
This include is not required, as it is included within file "fa.h" below

Reply:
Removed, thank you.